token/services/ttx/auditor.go

```diff
@@ -22,17 +22,31 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// TxStatus is the status of a transaction
+type TxStatus = ttxdb.TxStatus
```

Comment:
I see `token/services/auditor` is imported anyway so why not use what was already declared there and not copy it here?

Reply:
to use a single package `ttx` to do everything.